.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I guess this file should not be commit ;)

Reply:
![Alt Text](https://c.tenor.com/0VuncPcO_yIAAAAC/hide-the-simpsons.gif)